README.md

```diff
@@ -81,6 +81,33 @@ To get the full documentation run:
 $ ./marathon_lb.py --help
 ```
 
+### Providing SSL certificates
+You can provide your SSL certificate paths to be placed in frontend marathon_https_in section with `--ssl-certs`.
+
+``` console
+$ ./marathon_lb.py --marathon http://localhost:8080 --haproxy-config /etc/haproxy/haproxy.cfg --group external --ssl-certs /etc/ssl/site1.co,/etc/ssl/site2.co
+```
+
+If you are using the script directly, you have two options:
+
+ * Provide nothing and config will use `/etc/ssl/mesosphere.com.pem` as the certificate path. Put the certificate in this path or edit the file for the correct path.
+ * Provide --ssl-certs command line argument and config will use these paths.
+
+If you are using run.sh or Docker image, you have three options:
+
+ * Provide your certificate text in HAPROXY_SSL_CERT environment variable. Contents will be written to `/etc/ssl/mesosphere.com.pem`. Config will use this path unless you specified extra certificate paths as in the next option.
```

Comment:
I am using HAPROXY_SSL_CERT. How to get this going? I pasted the text of both my certificates. But only the first one seems to be picked up. Can you please help? @brndnmtthws  Thank you!